### ISSUE TITLE
winTile not compatible with GSConnect

### ISSUE THREAD

**Issue by sk0gen** [Fmstrat/wintile#21]
Problem with GSConnect extension. 

Steps To Reproduce:

1. Go to [GSConnect](https://extensions.gnome.org/extension/1319/gsconnect/)
2. Click on install
3. Error message: screenshots.

**Expected behavior**
Extension just works properly.

**Screenshots**

![image](https://user-images.githubusercontent.com/28295528/79632335-db0a0f80-815e-11ea-8fb5-35d00f000094.png)]
Error after installation of winTile.
![image](https://user-images.githubusercontent.com/28295528/79632493-ec075080-815f-11ea-8872-256745671228.png)]

**System Details (please complete the following information):**

 - **Installed from:** [GNOME Extensions Website]
 - **GNOME/Shell version:** [3.36]
 - **Distro/Release:** [Manjaro 19.02]

**Notes:
```js
window.gsconnect = {
	extdatadir: imports.misc.extensionUtils.getCurrentExtension().path,
	shell_version: parseInt(Config.PACKAGE_VERSION.split('.')[1], 10)
};
imports.searchPath.unshift(gsconnect.extdatadir);
```
[Code in winTile](https://github.com/Fmstrat/wintile/blob/8cdf9463b63708fa1418d418c06c2181744d3f9d/extension.js#L67-L71)

**Comment by ni-ka** [comment]
Indeed, this has been reported a few times now over at gs-connect https://github.com/andyholmes/gnome-shell-extension-gsconnect/issues/856 and https://github.com/andyholmes/gnome-shell-extension-gsconnect/issues/820

**Comment by gazhay** [comment]
I've installed this PR and it works to fix #29.
